modin/experimental/cloud/test/test_cloud.py

```diff
@@ -14,8 +14,88 @@
 import unittest.mock as mock
 import pytest
 from collections import namedtuple
+from inspect import signature
 from modin.experimental.cloud.rayscale import RayCluster
 from modin.experimental.cloud.cluster import Provider
+from ray.autoscaler.commands import (
+    create_or_update_cluster,
+    teardown_cluster,
+    get_head_node_ip,
+    _bootstrap_config,
+)
+
+
+def test__bootstrap_config():
+    sig_bootstrap_config = signature(_bootstrap_config)
+
+    def _bootstrap_config_mock(*args, **kwargs):
+        sig_bootstrap_config.bind(*args, **kwargs)
+        return {"empty": "config"}
+
+    with mock.patch(
+        "modin.experimental.cloud.rayscale._bootstrap_config", _bootstrap_config_mock
+    ):
+        RayCluster(Provider(name="aws"))
+
+
+def test_get_head_node_ip():
+    sig_get_head_node_ip = signature(get_head_node_ip)
+
+    def get_head_node_ip_mock(*args, **kwargs):
+        sig_get_head_node_ip.bind(*args, **kwargs)
+        return "127.0.0.1"
+
+    def bootstrap_mock(config):
+        config["auth"]["ssh_user"] = "modin"
+        config["auth"]["ssh_private_key"] = "X" * 20
+        return config
```

Comment:
these things look like they should be `pytest.fixture` (and other mocks used in other tests)